app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -68,19 +68,15 @@ $large-input-size: 50px;
   @include govuk-font($size: 19, $line-height: (28 / 19));
   margin: 0;
   width: 100%;
-  height: govuk-em(40, 16);
-  padding: govuk-em(6, 16);
+  height: govuk-em(40, 19);
```

Comment:
Just noticed that the problem you mentioned on search is appearing - can you take a look?

![Screenshot 2024-04-22 at 09 56 20](https://github.com/alphagov/govuk_publishing_components/assets/861310/b730c66f-59ee-4f4a-9ebb-7bd3c392b20d)

Reply:
This requries v5 to be in place, so we'll remove it from this PR and deploy it separately later.